tests/Test.ADAL.NET.Common/TestConstants.cs

```diff
@@ -39,6 +40,8 @@ public class TestConstants
         public static readonly string TenantSpecificAuthority = $"https://login.microsoftonline.com/{SomeTenantId}/";
         public static readonly string DefaultAuthorityGuestTenant = "https://login.microsoftonline.com/guest/";
         public static readonly string DefaultAuthorityCommonTenant = "https://login.microsoftonline.com/common/";
+        public static readonly string DisplayableIdUrl = DefaultAuthorityCommonTenant + "UserRealm/displayable@id.com";
```

Comment:
there is no such thing as displayableId Url. you should not hard code UPN in the url. consider having a function that takes in authority and UPN to create userrealm endpoint